### ISSUE TITLE
transmission-remote-gui version mismatch with remote transmission 

### ISSUE THREAD

**Issue by sSiDs** [transmission-remote-gui/transgui#1387]
I have upgraded transmission-daemon on Debian 10 to version 3.00+
but win version of Tr GUI shows still 2.94 on the left corner of programm window
---------------------------------------------------
sid@kino:~$ transmission-daemon --version
transmission-daemon 3.00+ (a7370fe8fb)
---------------------------------------------------
sid@kino:~$ systemctl status transmission-daemon.service 
● transmission-daemon.service - Transmission BitTorrent Daemon
     Loaded: loaded (/lib/systemd/system/transmission-daemon.service; enabled; vendor preset: enabled)
     Active: active (running) since Sat 2021-08-07 19:48:37 MSK; 1min 1s ago
   Main PID: 820 (transmission-da)
     Status: "Idle."
      Tasks: 4 (limit: 6942)
     Memory: 3.6M
     CGroup: /system.slice/transmission-daemon.service
             └─820 /usr/bin/transmission-daemon -f --log-error

Warning: some journal files were not opened due to insufficient permissions.
----------------------------------------------------------------------------------------

![tr](https://user-images.githubusercontent.com/20465150/128608089-e99a1267-a43e-480e-a7eb-15567855ef0d.jpg)

**Comment by quicktrick** [comment]
I have three Transmission servers: v2.94 on Qnap; v3.00 in a Docker container under openSUSE Tumbleweed; v3.00 in a jail under FreeBSD. Everywhere the version is displayed correctly.

**Comment by Mrnofish** [comment]
Yeah, this is in all likelihood some snafu on the daemon's side, which was probably not installed correctly, i.e. transmission-daemon was restarted using the old 2.94 binary.

This bug should be closed.
